packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -59,6 +59,26 @@ class LineSeriesLabel extends Label {
 type LineContext = SeriesNodeDataContext<LineNodeDatum>;
 type LineAnimationData = CartesianAnimationData<LineContext>;
 
+// TODO(olegat) move somewhere else?
+export class ErrorBarDatum {
```

Comment:
Nit: We usually reserve `Datum` as a term for actual data - I would probably just run with `ErrorBar` or `ErrorBarConfig` perhaps?

Reply:
ACK. I'll use `ErrorBarConfig`